tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/NoderealClient.sol

```diff
@@ -20,58 +20,49 @@ pragma solidity ^0.8.8;
 
 import "../libraries/Http.sol";
 import "../libraries/Utils.sol";
-
 library NoderealClient {
-    function getEthBalance(string memory url, string memory account)
-        internal
-        returns (bool, uint256)
-    {
-        HttpHeader[] memory headers = new HttpHeader[](0);
-        string memory request = string(
-            abi.encodePacked(
-                '{"jsonrpc": "2.0", "method": "eth_getBalance", "id": 1, "params": ["',
-                account,
-                '", "latest"]}'
-            )
-        );
-        (bool result, string memory balance) = Http.PostString(
-            url,
-            "/result",
-            request,
-            headers
-        );
-        if (result) {
-            return Utils.hexToNumber(balance);
-        } else {
-            return (false, 0);
-        }
-    }
-
-    function getErc20Balance(
-        string memory url,
-        string memory tokenContractAddress,
-        string memory account
-    ) internal returns (bool, uint256) {
-        HttpHeader[] memory headers = new HttpHeader[](0);
-        string memory request = string(
-            abi.encodePacked(
-                '{"jsonrpc": "2.0", "method": "nr_getTokenBalance20", "id": 1, "params": ["',
-                tokenContractAddress,
-                '","',
-                account,
-                '", "latest"]}'
-            )
-        );
-        (bool result, string memory balance) = Http.PostString(
-            url,
-            "/result",
-            request,
-            headers
-        );
-        if (result) {
-            return Utils.hexToNumber(balance);
-        } else {
-            return (false, 0);
-        }
-    }
+	function getTokenBalance(
```

Comment:
I wanted it to have 1 function per 1 rpc method but that's fine too.

Reply:
When using the two RPC methods, more "if" judgments need to be added. The code will look more complicated, so I merged them here.